Formula/hashlink.rb

```diff
@@ -0,0 +1,41 @@
+class Hashlink < Formula
+  desc "Virtual machine for Haxe"
+  homepage "https://hashlink.haxe.org/"
+  url "https://github.com/HaxeFoundation/hashlink/archive/1.11.tar.gz"
+  sha256 "b087ded7b93c7077f5b093b999f279a37aa1e31df829d882fa965389b5ad1aea"
+
+  depends_on "haxe" => :test
+  depends_on "jpeg-turbo"
+  depends_on "libogg"
+  depends_on "libpng"
+  depends_on "libuv"
+  depends_on "libvorbis"
+  depends_on "mbedtls"
+  depends_on "openal-soft"
+  depends_on "sdl2"
+
+  def install
+    system "make"
+    system "make", "install", "INSTALL_DIR=#{prefix}"
+  end
+
+  test do
+    haxebin = Formula["haxe"].bin
+    (testpath/"HelloWorld.hx").write <<~EOS
+      class HelloWorld {
+          static function main() Sys.println("Hello world!");
+      }
+    EOS
+
+    system "#{haxebin}/haxe", "-hl", "out.hl", "-main", "HelloWorld"
+    assert_equal "Hello world!\n", shell_output("#{bin}/hl out.hl")
+
+    system "#{haxebin}/haxelib", "newrepo"
+    system "#{haxebin}/haxelib", "install", "hashlink"
+    system "#{haxebin}/haxe", "-hl", "src/main.c", "-main", "HelloWorld"
+    system ENV.cc, "-O3", "-o", "hello-world",
+           "-std=c11", "-I", "src", "src/main.c",
+           "-L#{lib}", "-lhl"
```

Comment:
This is kinda cosmetic change but looks quite more readable after applying this:
```suggestion
    system ENV.cc, "-O3", "-std=c11", "-Isrc", "-I#{include}", "-L#{lib}", "-lhl",
                   "src/main.c", "-o", "hello-world"
```